### ISSUE TITLE
Wording in MA text

### ISSUE THREAD

**Issue by ARMediting** [gryphonshafer/Quizzing-Rule-Book#215]
This is the current text about Multiple Answers:

> 2.2.2.2. Multiple Answer (MA)
> Must meet all the requirements of an INT except for the single answer requirement
> Must have more than 1 answer
> The clarification of a single answer does not meet the above requirement
> The identification of a singular pronoun does not meet the above requirement
> The identification of a plural pronoun does meet the above requirement if there is more
> than 1 clarification of the pronoun

Since the prompt "Identify" has returned to "Clarify," should we reword the above descriptions? Change "clarification" to "explanation" or the like, and "identification" to "clarification," since that's how a quizmaster would likely prompt during such a question?

**Comment by kclimenhaga** [comment]
I agree that "identification" should be changed to "clarification". I don't think we should change "clarification" to "explanation", though. My understanding of that rule is that it would be for a situation like "James, the brother of John", where "the brother of John" is a clarification of the single answer "James". I wouldn't call that an explanation. I am open to considering how we can make the explanation more clear in general.

**Comment by ARMediting** [comment]
What about "Additional information about a singular answer"?
